templates/userdata.tpl

```diff
@@ -2,3 +2,29 @@
 set -o xtrace
 
 /etc/eks/bootstrap.sh --apiserver-endpoint '${cluster_endpoint}' --b64-cluster-ca '${certificate_authority_data}' ${bootstrap_extra_args} --kubelet-extra-args '${kubelet_extra_args}' '${cluster_name}'
+
+enable_cloudwatch="${var.enable_cloudwatch}"
+
+if [ "$enable_cloudwatch" = true ]; then
```

Comment:
Although not strictly necessary it's good practice to quote all strings to make them distinguishable from keywords.